mobile/src/lib/cozy-helper.js

```diff
@@ -34,3 +34,16 @@ export const initBar = () => {
     lang: 'en'
   })
 }
+
+export const isClientRegistered = async (client) => {
+  return await cozy.client.auth.getClient(client).then(client => true).catch(() => false)
```

Comment:
When the device is offline, `isClientRegistered` will return false. I'd have made the opposite supposition: that the client is still registered.